hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/schema/BaselineMetricAggregationContainer.java

```diff
@@ -0,0 +1,57 @@
+package org.hypertrace.graphql.metric.schema;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import org.hypertrace.core.graphql.common.schema.time.TimeUnit;
+import org.hypertrace.graphql.metric.schema.argument.MetricAggregationAvgRateSizeArgument;
+import org.hypertrace.graphql.metric.schema.argument.MetricAggregationAvgRateUnitsArgument;
+import org.hypertrace.graphql.metric.schema.argument.MetricAggregationPercentileSizeArgument;
+
+import static org.hypertrace.graphql.metric.schema.MetricAggregationContainer.*;
+
+public interface BaselineMetricAggregationContainer {
```

Comment:
Can we make MetricAggregationContainer an interface to make sure the two variants of it stay in sync?